downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -54,6 +54,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.9.0%22+label%3AType%2FBug).
 
+### New features
```

Comment:
Should be the first section under "Ballerina library updates"?

Reply:
Fixed in [e264ba8](https://github.com/ballerina-platform/ballerina-dev-website/pull/8892/commits/e264ba8234166595d22d42a38ff92cc267238fa9)